common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -7,6 +7,15 @@ param(
 
 . (Join-Path $PSScriptRoot ../../../eng/scripts/smoketest/Get-Package-Version.ps1)
 
+# Define a set that contains the list of packages which should not be included
+# as references to the smoke tests.  This is necessary because some compatibility
+# packages do not target netstandard2.0, and the package source is unable to
+# filter upstream to restrict to the appropriate packages for the running target.
+
+$packageExcludeSet = [System.Collections.Generic.HashSet[string]]@(
```

Comment:
Using a hash table would have allowed using pure PowerShell here, but it seemed wrong to use it when I only care about the keys.

Reply:
I mean, given we probably need so few packages here anyway, even a simple array (just elide the type conversion) would be fine. It's unlikely someone would dup an item and, even if they did, we're talking about ms here, not even seconds.

Reply:
Fair point.   Converted to an array.